azure-pipelines.yml

```diff
@@ -41,15 +41,30 @@ steps:
     dockerHubImageName: $(imageName)
     CC_TEST_REPORTER_ID: $(ccReporterID)
 - script: |
+    set -x
```

Comment:
adding that kinda feels like this whole script block should be in a bash file

Reply:
Yeah possibly - if we were to do this we would then need to move other qualifying script blocks in the azure-pipeline.yml to script blocks to. 

Personally I think it may make it more readable if it's done inline. 
@misaka @rizzkhan1 what do you think?

Reply:
I don't understand. These are essentially shoved into a bash script by the pipeline processor, are you saying we should have a separate bash script for each job here?

Reply:
I think we can clean this up by re-adding the `.rspec_parallel` file, maybe as `.rspec_parallel_azure`, that would get rid of 50% of the lines here I think.